lascli/parser/workflows.py

```diff
@@ -73,6 +73,11 @@ def create_workflows_parser(subparsers):
         type=json_path,
         help='path to the execution completed configuration for the workflow',
     )
+    create_workflow_parser.add_argument(
+        '--metadata', 
+        type=json_or_json_path, 
+        help='Add additional custom information about the workflow',
```

Comment:
Should help say anything about the fact that metadata needs to be a json file or given in json format? Here and also below.
```
help='Add additional custom information about the workflow (through json)',
```